test/command.js

```diff
@@ -61,8 +61,8 @@ describe('Command', () => {
       argv['baz-qux'].should.equal('world')
     })
 
-    it('populates argv with camel-case variants of variadic args when possible', () => {
-      const argv = yargs('foo hello world !')
+    it('populates argv with camel-case variants of variadic args when possible', async () => {
```

Comment:
adding some specific tests for chains of `async` middleware, would be good, I'd expect this to work like:

```
for (const middleware of stack) {
  await middleware;
  await command;
}
```

Reply:
Oops! I forgot this point. I will add some tests.

Reply:
done